src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -46,18 +47,18 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options, IEnu
             ReloadLoggerOptions(options.CurrentValue);
             _optionsReloadToken = _options.OnChange(ReloadLoggerOptions);
 
-            _messageQueue = new ConsoleLoggerProcessor();
-
+            IConsole? console, errorConsole = null;
```

Comment:
```suggestion
            IConsole? console;
            IConsole? errorConsole;
```

1. The variables are unconditionally set, so there is no reason to initialize them to `null`.
2. Combining variable declarations is harder to read.